csrankings.csv

```diff
@@ -16383,4 +16383,21 @@ Zygmunt J. Haas,University of Texas at Dallas,http://www.utdallas.edu/~zjh130030
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Øystein Nytrø,NTNU,https://www.ntnu.edu/employees/nytroe,-Zavs80AAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
-Liqing Zhang 0001,Shanghai Jiao Tong University,http://bcmi.sjtu.edu.cn/~zhangliqing/Index.html,1smFmxAAAAAJ
+Liqing Zhang 0001,Shanghai Jiao Tong University,http://bcmi.sjtu.edu.cn/~zhangliqing/Index.html,1smFmxAAAAAJ
+Yanyan Zhao,Harbin Institute of Technology,http://homepage.hit.edu.cn/yanyan,NOSCHOLARPAGE
+Ming Liu 0004,Harbin Institute of Technology,http://homepage.hit.edu.cn/liuming1981,VJtmTREAAAAJ
+Weinan Zhang,Harbin Institute of Technology,http://ir.hit.edu.cn/~wnzhang/,DBLdEf4AAAAJ
```

Comment:
Does not appear to be in the CS department.

Reply:
Please go through and remove all faculty not in the CS department.

Reply:
I have gone through the list and removed two more teachers. I think the file you attached in your  comment is not the latest version because I have removed Yanyan Zhao earlier. You can see the files changes from all commits.